src/mame/drivers/qx10.cpp

```diff
@@ -48,6 +48,7 @@
 #include "machine/ram.h"
 #include "machine/upd765.h"
 #include "machine/z80sio.h"
+#include "bus/epson_qx/option.h"
```

Comment:
Please put this with the other included headers from src/devices/bus – it makes keeping track of stuff easier.